code/modules/mob/living/carbon/human/human_defense.dm

```diff
@@ -65,7 +65,10 @@
 					P.firer = src
 					P.yo = new_y - curloc.y
 					P.xo = new_x - curloc.x
-					P.Angle = null
+					var/new_angle_s = P.Angle + rand(120,240)

```

Comment:
@kevinz000 is there any helpers for reflections?

Reply:
@kevinz000

Reply:
I can answer and say I already looked and there isn't, otherwise he would have used them in his ricochet code or reflector code.

Reply:
hi I'm back lemme search

Reply:
https://github.com/tgstation/tgstation/blob/edef11f0944e1365f28a30489420f8f0eba8e327/code/game/objects/structures/reflector.dm#L200

search harder codeslave

Reply:
actually shit I have no idea how this'll interact with eshields as they don't have a direction other than just the person's direction, use the double reflector code i guess.

Reply:
Use the helper please

Reply:
ITT we overcomplicate a 4-line fix by forcing me to use some stupid define that isn't even appropriate for this sort of behavior.

GET_ANGLE_OF_INCIDENCE doesn't mesh with E-shield or construct behavior AT ALL.

The whole point of both is to send projectiles back at the shooter's direction, forcing me to account for the target's direction serves WHAT purpose again?